microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,54 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
 
 
 <!-- ====================================================================== -->
+## 1.0.1369-prerelease

```

Comment:
@nishitha-burman Is the overview for the api articles overview need to be updated.  Will it be in a seperate pr?    Will we need to update it?

Reply:
Yes, that will be in a separate PR. @imicy

Reply:
Sweet.  Please add me to the pr.